website/content/docs/internals/plugins/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+page_title: Plugin System
+description: Learn about Vault's plugin system.
+---
+
+
+# Plugin System
+
+All Vault auth and secret backends are considered plugins. This simple concept
```

Comment:
The terms `auth backends` and `secret backends` are legacy terms. We should be using `auth methods` and `secrets engines`. 

Feel free to ignore if you disagree. 

```suggestion
All Vault auth methods and secrets engines are considered plugins. This simple concept
```